app/views/layouts/application.html.erb

```diff
@@ -34,7 +34,7 @@
       </script>
     <% end %>
 
-    <% if Rails.env.production? %>
+    <% if Rails.env.production? && ENV["APP_HOST"].present? %>
```

Comment:
Why was this added?

Reply:
Stops you from mocking prod env in tests in Feature flags otherwise

Reply:
Can't we just provide a mock value in tests? Seems cleaner than changing the actual code here for something completely unrelated.